docs/satellite-clone.md

```diff
@@ -91,3 +91,24 @@ On the target server:
       # ansible-playbook satellite-clone-playbook.yml
     ```
   The playbook run may may take a while to complete.
+
+## Cloning a Satellite With Remote Databases
+
+Note: The backup used for these procedures must have the correct options to manage remote databases in `/etc/foreman-installer/scenarios.d/satellite-answers.yaml`. Please check that the answers file has the correct database management values filled in the configuration backup file. These will be values such as `db-manage` in the `foreman` section and `pulp-manage-db` in the `katello` section. They are currently (at the time of this writing) [missing from the initial setup documentation](https://bugzilla.redhat.com/show_bug.cgi?id=1887846)
+
+1. Set up the remote databases according to [Satellite documentation](https://access.redhat.com/documentation/en-us/red_hat_satellite/) in the same way that they were set up on your original Satellite.
+2. Set up a base RHEL7 server that will become the cloned satellite and make sure the target clone server has network access to the remote databases.
```

Comment:
Step 2 and 1 should swap places because current 1 implies that would-be satellite server already exists

Reply:
I think the big part that is missing is to not install Satellite, just set up the remote dbs and a blank RHEL7 server. I'll find a better way to word this!